models/bamf_ct_liver_tumor/meta.json

```diff
@@ -0,0 +1,136 @@
+{
+  "id": "",
+  "name": "bamf_ct_liver_tumor",
+  "title": "BAMF CT Liver and Tumor Seg (nnU-Net)",
+  "summary": {
+    "description": "An nnU-Net based model to segment liver and tumor from CT scans",
+    "inputs": [
+      {
+        "label": "Input Image",
+        "description": "The CT scan of a patient.",
+        "format": "DICOM",
+        "modality": "CT",
+        "bodypartexamined": "LIVER",
+        "slicethickness": "2.5mm",
+        "non-contrast": false,
+        "contrast": true
+      }
+    ],
+    "outputs": [
+      {
+        "label": "Segmentation",
+        "type": "Segmentation",
+        "description": "Segmentation liver",
+        "classes": [
+          "LIVER",
+          "LIVER+TUMOR"
```

Comment:
Should be `LIVER+NEOPLASM_MALIGNANT` to be consistent.
Could you briefly elaborate why we cannot use the `TUMOR` label from SegDB?

Reply:
Thanks for adding segdb entries. I have changed this to `LIVER+NEOPLASM`.